### PR TITLE
Train without the validation set

### PR DESCRIPTION
**Pull request by Eleven1Liu**
**Description** Train without validation when `val_size` is set to 0.

### CODE REVIEW

File: torch_trainer.py [hunk 1]
```diff
@@ -126,13 +126,20 @@ def train(self):
...
+        else:
+            val_loader = self._get_dataset_loader(split='val')
+            self.trainer.fit(self.model, train_loader, val_loader)
+
+        # Set model to the best model. If the validation process is skipped during
+        # training (i.e., val_size=0), the model is set to the last model.
         checkpoint_callback = [callback for callback in self.trainer.callbacks if isinstance(callback, ModelCheckpoint)][0]
-        logging.info(f'Finished training. Load best model from {checkpoint_callback.best_model_path}')
-        self._setup_model(checkpoint_path=checkpoint_callback.best_model_path)
+        model_path = checkpoint_callback.best_model_path if checkpoint_callback.best_model_path else checkpoint_callback.last_model_path
```
Comment (sian-chen) [inline torch_trainer.py:140]:
```suggestion
        model_path = checkpoint_callback.best_model_path or checkpoint_callback.last_model_path
```
This is a more pythonic way to write `A if A else B`.

**Review by sian-chen — APPROVED** [review]

File: libmultilabel/nn/data_utils.py [hunk 2]
```diff
@@ -122,7 +122,7 @@ def load_datasets(
 ):
     """Load data either from the specified data paths (i.e., `train_path`, `test_path`, and `val_path`)
     or from the data files (i.e., `train.txt`, `test.txt`, and `valid.txt`) in the data directory.
-    If `valid.txt` does not exist, the validation set will automatically split from the training dataset.
+    If `valid.txt` does not exist, the validation set will automatically split from the training dataset if val_size is not 0.
```
Comment (cjlin1) [inline libmultilabel/nn/data_utils.py:125]: If `valid.txt` does not exist but val_size > 0, the validation set will be split from the training dataset.

Comment (Eleven1Liu) [reply libmultilabel/nn/data_utils.py:125]: Resolved in PR #109
